v3/library-install.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+# Necessary to stop pull requests from forks from running.
+if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
+  npm install
+  npm install --no-save blackbaud/skyux-builder-config#visual-scripts
```

Comment:
Note: remove `#visual-scripts` after this is branch is merged: https://github.com/blackbaud/skyux-builder-config/pull/22